### DISCUSSION TITLE
Localized plot

### DISCUSSION THREAD

**Discussion by metallarodark** [cinemagoer/cinemagoer#464] [Q&A]
Hello, I wonder if is it possible to get localized plot. I'm in Italy. 

`from imdb import Cinemagoer

ia = Cinemagoer('https', languages='it-IT')

movie_id = input("Insert ID: ")

movie = ia.get_movie(movie_id)

plot = movie.get('plot')

print(plot[0])`

This prints the english plot while I can see that the italian one is available on the site switching the language. Am I doing anything wrong? Is it not possible at the moment? Can it be added in the future?

Thank you!

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
